code/modules/reagents/chemical_research/research-events.dm

```diff
@@ -0,0 +1,58 @@
+GLOBAL_DATUM_INIT(ddi_experiment, /datum/research_event/ddi_experiment, new)
+#define MAX_POINTS 20
+
+/datum/research_event/ddi_experiment
+	var/DDI_experiment_triggered = FALSE
+	var/mob/living/carbon/xenomorph/linked_xeno = null
+	var/client/player = null
+	var/timer
+	var/total_points_given = 0
+
+/datum/research_event/ddi_experiment/proc/get_mind(mob/living/new_xeno)
+	SIGNAL_HANDLER
+	player = new_xeno.client
+
+/datum/research_event/ddi_experiment/proc/trigger(xeno)
+	if(DDI_experiment_triggered)
+		return
+	DDI_experiment_triggered = TRUE
+	linked_xeno = xeno
+	if(linked_xeno.client)
+		player = linked_xeno.client
+	RegisterSignal(linked_xeno, COMSIG_MOB_NEW_MIND, PROC_REF(get_mind)) //this works as long as the client is not forcefully set by admins
+
+	ai_announcement("Notice: Unidentified lifesign detected at research containment created through DNA disintegration, analyzing data...")
```

Comment:
DNA dising is a top secret stuff its impossible that ares (and old USCM AI) knows about it.
Change it to something like ...
"Unknown lifesign detected in contaiment .... Analizing"